resource/airflow_dag_pipeline.yaml

```diff
@@ -174,7 +174,7 @@ jobs:
             - |
               cp -R ./pr-with-iam/* ./pull-request-edit
               jq '.role_name = "airflow_test_dag_role"' ./pr-with-iam/deploy.json > ./pull-request-edit/deploy.json
-              jq '.Statement += [{"Sid": "denyall","Action": ["*"],"Effect": "Deny","Resource": ["*"]}]' ./pr-with-iam/iam_policy.json > ./pull-request-edit/iam_policy.json
```

Comment:
What was the problem with the Sid in this statement?

Reply:
If someone has a policy with another sid of the same name it fails. Best to have no sid so there will not be conflicts

Reply:
👍